### PR TITLE
Add a description for the `$method` modifier

### PR DESCRIPTION
**Pull request by ngorskikh**

### CODE REVIEW

**Comment by sfionov** [comment]
@ngorskikh Can I write `$method=GET|POST`? And `$method=GeT|pOsT` ?

File: docs/general/ad-filtering/create-own-filters.md [hunk 1]
```diff
@@ -1006,6 +1007,20 @@ If you want the rule not to be applied to certain apps, start the app name with
...
 > Only AdGuard for Windows, Mac, Android are technically capable of using rules with `$app` modifier.
 
+#### **`$method`** {#method-modifier}
+
+This modifier limits the scope of the rule to requests that use the specified set of HTTP methods. Negated methods are allowed. The set of allowed methods is the same as in Chrome's [declarativeNetRequest](https://developer.chrome.com/docs/extensions/reference/declarativeNetRequest/#type-RequestMethod).
+
+**Examples**
+
+* `||evil.com^$method=get|head` blocks only GET and HEAD requests to `evil.com`.
+* `||evil.com^$~post|~put` blocks any requests except POST or PUT to `evil.com`.
```
Comment (sxgunchenko) [inline docs/general/ad-filtering/create-own-filters.md:1017]:
```suggestion
* `||evil.com^$method= ~post|~put` blocks any requests except POST or PUT to `evil.com`.
```

**Review by sxgunchenko — CHANGES_REQUESTED** [review]

File: docs/general/ad-filtering/create-own-filters.md [hunk 2]
```diff
@@ -1006,6 +1007,20 @@ If you want the rule not to be applied to certain apps, start the app name with
 >
 > Only AdGuard for Windows, Mac, Android are technically capable of using rules with `$app` modifier.
 
+#### **`$method`** {#method-modifier}
+
+This modifier limits the scope of the rule to requests that use the specified set of HTTP methods. Negated methods are allowed. The set of allowed methods is the same as in Chrome's [declarativeNetRequest](https://developer.chrome.com/docs/extensions/reference/declarativeNetRequest/#type-RequestMethod).
```
Comment (slavaleleka) [inline docs/general/ad-filtering/create-own-filters.md:1012]: will it work only in chrome? and only if chrome v91 or newer?

**Review by sxgunchenko — APPROVED** [review]

File: docs/general/ad-filtering/create-own-filters.md (hunk 2 above)
Comment (ngorskikh) [reply docs/general/ad-filtering/create-own-filters.md]: No, only the what you can type after `method=` is defined by this list :) We are still thinking about lifting this restriction, please ignore for now

**Review by slavaleleka — APPROVED** [review]

**Review by el-termikael — APPROVED** [review]

**Review by sxgunchenko — APPROVED** [review]

**Review by ameshkov — APPROVED** [review]

**Review by el-termikael — APPROVED** [review]

**Pull request merged**